vcx/wrappers/node/src/api/wallet.ts

```diff
@@ -329,12 +357,21 @@ export class Wallet {
   }
 
   /**
-   * @memberof Wallet
-   * @description Adds tags to a record already in the wallet
-   * @static
-   * @async
-   * @param {Record} record
-   * @returns {Promise<void>}
+   * Adds tags to a record already in the wallet
+   *
+   * Example:
+   * ```
+   * await Wallet.addRecord({
+   *    id: 'RecordId',
+   *    tags: {},
+   *    type_: 'TestType',
+   *    value: 'RecordValue'})
+   *
+   * addRecordTags({  id: 'RecordId',
+   *     tags: { "tagName1": "tag value 1", "~tagName2": "tag value 2 unencrypted", "tagName3", 1},
```

Comment:
Due to such a long line it should be formatted with each parameter being on its own line. Maybe this was auto generated?